test/fixtures/service.fixtures.js

```diff
@@ -154,7 +154,7 @@ module.exports = {
 
     if (opts.default_billing_address_country !== null) {
       service.default_billing_address_country = opts.default_billing_address_country === undefined
-        ? 'GB'
+        ? 'GB2 - ' + JSON.stringify(opts.default_billing_address_country)
```

Comment:
I'm curious about what this does? will this not make the response `GB2 - undefined`?

Reply:
This was a debugging moment that shouldn't have made it into the PR.  Good spot, we'll remove it.